src/env-inl.h

```diff
@@ -303,11 +303,11 @@ inline Environment* Environment::GetCurrent(v8::Isolate* isolate) {
 
 inline Environment* Environment::GetCurrent(v8::Local<v8::Context> context) {
   if (UNLIKELY(context.IsEmpty() ||
-      context->GetNumberOfEmbedderDataFields() <
-          ContextEmbedderIndex::kContextTag ||
-      context->GetAlignedPointerFromEmbedderData(
-          ContextEmbedderIndex::kContextTag) !=
-          Environment::kNodeContextTagPtr)) {
+               context->GetNumberOfEmbedderDataFields() <=
+                   ContextEmbedderIndex::kContextTag ||
+               context->GetAlignedPointerFromEmbedderData(
+                   ContextEmbedderIndex::kContextTag) !=
+                   Environment::kNodeContextTagPtr)) {
```

Comment:
Since you're here: I was looking at this code this morning and I think it'd be a lot easier to read as a bunch of if guards:
```c++
if (context.IsEmpty()) return nullptr;
if (context->GetNumberOfEmbedderDataFields() <= ContextEmbedderIndex::kContextTag)
  return nullptr;
// ...
```